homeassistant/components/velbus/const.py

```diff
@@ -14,6 +14,7 @@
 CONF_CONFIG_ENTRY: Final = "config_entry"
 CONF_INTERFACE: Final = "interface"
 CONF_MEMO_TEXT: Final = "memo_text"
+CONF_TLS: Final = "tls"
```

Comment:
Why aren't you using `CONF_SSL` from the common const?

Reply:
as the velbus modules use the term tls everywhere, so its more clear for the user